packages/ChangesReloaded-Core.package/CRBrowser.class/instance/changeListItemsFor..st

```diff
@@ -0,0 +1,4 @@
+accessing
+changeListItemsFor: aSymbol
+
+	^ self methodHistories at: aSymbol ifAbsent: [OrderedCollection new]
```

Comment:
methodHistories und aSymbol sind nicht kongruent in ihrer Benennung (methodHistories -> histories)

Reply:
veraltet